go/src/koding/klientctl/endpoint/team/team.go

```diff
@@ -40,6 +47,23 @@ func (c *Client) Used() *Team {
 	return &c.used
 }
 
+// List returns the list of user's teams.
+func (c *Client) List(opts *ListOptions) ([]*team.Team, error) {
+	c.init()
+
+	req := &stack.TeamListRequest{}
+	if opts != nil {
+		req.Slug = opts.Slug
+	}
+
+	resp := &stack.TeamListResponse{}
+	if err := c.kloud().Call("team.list", req, resp); err != nil {
```

Comment:
Out of curiosity, question about `var resp stack.TeamListResponse` over `resp := &stack.TeamListResponse{}` - why not:

```
var resp stack.TeamListResponse

...

if err := c.kloud().Call("team.list", req, &resp); err != nil {
...
```

This way the code is more readable imho, you see what var is being written to.